homeassistant/components/overkiz/button.py

```diff
@@ -0,0 +1,81 @@
+"""Support for Overkiz (virtual) buttons."""
+from __future__ import annotations
+
+from homeassistant.components.button import ButtonEntity, ButtonEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ENTITY_CATEGORY_DIAGNOSTIC
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import HomeAssistantOverkizData
+from .const import DOMAIN, IGNORED_OVERKIZ_DEVICES
+from .entity import OverkizDescriptiveEntity
+
+BUTTON_DESCRIPTIONS: list[ButtonEntityDescription] = [
+    # My Position (cover, light)
+    ButtonEntityDescription(
+        key="my",
+        name="My Position",
+        icon="mdi:star",
+    ),
+    # Identify
+    ButtonEntityDescription(
+        key="identify",  # startIdentify and identify are reversed... Swap this when fixed in API.
+        name="Start Identify",
+        icon="mdi:human-greeting-variant",
+        entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
+        entity_registry_enabled_default=False,
+    ),
+    ButtonEntityDescription(
+        key="stopIdentify",
+        name="Stop Identify",
+        icon="mdi:human-greeting-variant",
+        entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
+        entity_registry_enabled_default=False,
+    ),
+    ButtonEntityDescription(
+        key="startIdentify",  # startIdentify and identify are reversed... Swap this when fixed in API.
+        name="Identify",
+        icon="mdi:human-greeting-variant",
+        entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
+    ),
+]
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+):
+    """Set up the Overkiz button from a config entry."""
+    data: HomeAssistantOverkizData = hass.data[DOMAIN][entry.entry_id]
+    entities: list[ButtonEntity] = []
+
+    supported_commands = {
+        description.key: description for description in BUTTON_DESCRIPTIONS
+    }
+
+    for device in data.coordinator.data.values():
+        if (
+            device.widget not in IGNORED_OVERKIZ_DEVICES
+            and device.ui_class not in IGNORED_OVERKIZ_DEVICES
+        ):
```

Comment:
If you reverse the condition and `continue`, you can save some indent

Reply:
Will have a look tomorrow 👍🏼. And fix that for all other entities as well. 
(or I can do it in the another PR which has the same base)

Reply:
Since its going to be the same for the existing platforms that are already merged, I'll go ahead and merge this now, and it can be adjusted in the next PR since its not a material change anyways.